packages/ndla-ui/src/locale/messages-nn.ts

```diff
@@ -951,6 +951,12 @@ const messages = {
           shared:
             "No kan du dele denne lenka med elevar eller andre lærarar. Dersom du gjer endringar i mappa, blir dei synlege for alle du har delt lenka med.",
         },
+        warning: {
+          authenticated:
+            "Denne mappa er delt av {{ name }}, og inneheld fagstoff, oppgåver og lenkjer til tekstar frå både NDLA og andre nettstader.",
+          unauthenticated:
+            "Denne mappa er delt av {{ name }}, og inneheld fagstoff, oppgåver og lenkjer til tekstar frå både NDLA og andre nettstader. Logg inn på Min NDLA for å kopiera mappa eller lager lenkja.",
```

Comment:
Lurt å bruke samme forma overalt.
```suggestion
            "Denne mappa er delt av {{ name }}, og inneheld fagstoff, oppgåver og lenker til tekstar frå både NDLA og andre nettstader.",
          unauthenticated:
            "Denne mappa er delt av {{ name }}, og inneheld fagstoff, oppgåver og lenker til tekstar frå både NDLA og andre nettstader. Logg inn på Min NDLA for å kopiera mappa eller lager lenka.",
```